### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by zaneadix**
Started the documentation effort. Just writing everything out in a README within the pokemon_v2 directory for now. Currently some things have been changed that should not have been. I will work this out.

### CODE REVIEW

File: config/settings.py [hunk 1]
```diff
@@ -89,7 +89,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'pokeapi_co_db',
```
Comment (phalt) [inline config/settings.py:92]: This needs to remain as `pokeapi_co_db` for the live site. Perhaps a debug setting?

**Comment by phalt** [comment]
This is looking good. I would add in an example request and response for each resource if you could, one that can be copy and pasted :+1:

**Comment by zaneadix** [comment]
https://github.com/zaneadix/pokeapi/tree/docs/pokemon_v2

@phalt I think this should be good. I added example responses with all arrays reduced to a single index to keep things as short as possible. Any thoughts on what else could be added?

File: pokemon_v2/serializers.py [hunk 2]
```diff
@@ -1036,6 +1037,28 @@ def get_ability_pokemon(self, obj):
 
         return pokemon
 
+    # def get_flavor_text(self, obj):
```
Comment (phalt) [inline pokemon_v2/serializers.py]: Is this supposed to be part of this PR?

Comment (zaneadix) [inline pokemon_v2/serializers.py]: Whoops nope. This was an experiment that seemed not necessary in the end. Thanks for catching. Took it out.

**Pull request merged**